src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -6076,96 +6097,162 @@ void ObjectMgr::LoadAccessRequirements()
     }
-
-    //                                               0      1           2          3          4     5      6             7             8                      9                  10
-    QueryResult result = WorldDatabase.Query("SELECT mapid, difficulty, level_min, level_max, item, item2, quest_done_A, quest_done_H, completed_achievement, quest_failed_text, item_level FROM access_requirement");
-    if (!result)
+    //                                                               0       1            2           3          4            5
+    QueryResult access_template_result = WorldDatabase.Query("SELECT id, map_id, difficulty, min_level, max_level, min_avg_item_level FROM dungeon_access_template");
+    if (!access_template_result)
     {
-        sLog->outString(">> Loaded 0 access requirement definitions. DB table `access_requirement` is empty.");
+        sLog->outString(">> Loaded 0 access requirement definitions. DB table `dungeon_access_template` is empty.");
         sLog->outString();
         return;
     }
 
     uint32 count = 0;
+    uint32 countProgressionRequirements = 0;
 
     do
     {
-        Field* fields = result->Fetch();
+        Field* fields = access_template_result->Fetch();
 
-        ++count;
-
-        uint32 mapid = fields[0].GetUInt32();
-        uint8 difficulty = fields[1].GetUInt8();
-        uint32 requirement_ID = MAKE_PAIR32(mapid, difficulty);
-
-        AccessRequirement* ar = new AccessRequirement();
+        //Get the common variables for the access requirements
+        uint8 dungeon_access_id = fields[0].GetUInt8();
+        uint32 mapid            = fields[1].GetUInt32();
+        uint8 difficulty        = fields[2].GetUInt8();
 
-        ar->levelMin                 = fields[2].GetUInt8();
-        ar->levelMax                 = fields[3].GetUInt8();
-        ar->item                     = fields[4].GetUInt32();
-        ar->item2                    = fields[5].GetUInt32();
-        ar->quest_A                  = fields[6].GetUInt32();
-        ar->quest_H                  = fields[7].GetUInt32();
-        ar->achievement              = fields[8].GetUInt32();
-        ar->questFailedText          = fields[9].GetString();
-        ar->reqItemLevel             = fields[10].GetUInt16();
+        //Set up the access requirements
+        DungeonProgressionRequirements* ar = new DungeonProgressionRequirements();
+        ar->levelMin     = fields[3].GetUInt8();
+        ar->levelMax     = fields[4].GetUInt8();
+        ar->reqItemLevel = fields[5].GetUInt16();
 
-        if (ar->item)
+        //                                                                  0                 1               2                 3        4
+        QueryResult progression_requirements_results = WorldDatabase.PQuery("SELECT requirement_type, requirement_id, requirement_hint, priority faction FROM dungeon_access_requirements where dungeon_access_id = %u", dungeon_access_id);
```

Comment:
```suggestion
        QueryResult progression_requirements_results = WorldDatabase.PQuery("SELECT requirement_type, requirement_id, requirement_hint, priority, faction FROM dungeon_access_requirements where dungeon_access_id = %u", dungeon_access_id);
```

missing `,` critical issue :P @peti446